packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.types.ts

```diff
@@ -64,6 +64,11 @@ export interface ISearchBoxProps extends React.InputHTMLAttributes<HTMLInputElem
    */
   ariaLabel?: string;
 
+  /**
+   * The props for the clear button.
+   */
+  clearButtonProps?: ISearchBoxClearButtonProps;
```

Comment:
This should just be `IButtonProps` which should support whatever a button supports. That allows the caller to do things like put a `data-automation-id` on it.

Reply:
I'd considered that but thought I'd start with the conservative approach of whitelisting properties. Is this the last blocking issue?

Reply:
To clarify: I will change this to ibuttonprops tomorrow, I'm just asking if there are other changes needed as well.

Reply:
Done.